src/arch/x86/object/vcpu.c

```diff
@@ -290,7 +290,14 @@ static bool_t BOOT_CODE init_vtx_fixed_values(bool_t useTrueMsrs)
     entry_control_high = x86_rdmsr_low(entry_ctls);
     entry_control_low = x86_rdmsr_high(entry_ctls);
 
-    cr0_high = x86_rdmsr_low(IA32_VMX_CR0_FIXED0_MSR);
+    // Intel SDM, Vol. 3C, 24.8:
+    //
+    // > Later processors support a VM-execution control called "unrestricted
+    // > guest" (see Section 25.6.2). If this control is 1, CR0.PE and CR0.PG
+    // > may be 0 in VMX non-root operation (even if the capability MSR
+    // > IA32_VMX_CR0_FIXED0 reports otherwise).1 Such processors allow guest
+    // > software to run in unpaged protected mode or in real-address mode.
```

Comment:
The numbers don't match the combined SDM I just downloaded from https://www.intel.com/content/www/us/en/developer/articles/technical/intel-sdm.html. Could you specify which document exactly or double check they didn't change the chapters is the newest version or something?

In my version that text is in chapter 25.8, "Restrictions on VMX Operations". Considering the chapters change, perhaps also give the chapter title so it's easier to find it.